bool enableRecognition
bool enableSkeleton
bool enableGestures
bool enableLandmarks
bool enableHeadBoundingBox
bool enableHeadPose
---
bool status
